src/index.js

```diff
@@ -307,13 +307,13 @@ exports.init = async (Implementation) => {
         configStore.integrator,
         configStore.lianaOptions,
       ).perform();
-      new ActionsRoutes(
+      new ActionsRoutes().perform(
```

Comment:
I'm not sure a route should be injected in `src/index.js`. I checked in forestadmin-server and it seems It's not the case. So `ActionsRoutes` use DI internally but it is not injectable (since it should not be injected). I prefer not changing the `src/index.js` current design. It's just a cautious choice, not a design choice.

Reply:
👍